src/utils/assign.js

```diff
@@ -0,0 +1,13 @@
+export default function assign(obj, ...sources) {
```

Comment:
but actually i think it'd be better to use the assign function from lodash or some other library rather than maintaining our own assign method